tracer/src/Datadog.Trace/Ci/Agent/MessagePack/SpanEventMessagePackFormatter.cs

```diff
@@ -14,6 +14,7 @@ public override int Serialize(ref byte[] bytes, int offset, SpanEvent value, IFo
         {
             if (value is null)
             {
+                Log.Warning("SpanEventMessagePackFormatter: SpanEvent is null");
```

Comment:
I take it this should never happen? Though makes me wonder if it has 🤔 Just don't want to spam the logs if it could happen in "normal" situations

Reply:
should never happen but is not fatal, that's why I'm logging it as a Warning.